sysutils/snapraid/Portfile

```
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        amadvance snapraid 12.3 v
+
+name                snapraid
+version             12.3
+categories          sysutils
+
+license             GPL-3
+maintainers         {anquieta @anquieta}
```

Ports are additionally `openmaintainer` unless there is a reason why you and only you should be allowed to make changes or updates to this port:

```suggestion
maintainers         {anquieta @anquieta} \
                    openmaintainer
```

---

also, you should add your (obfuscated) e-mail address as what you use now implies a ...@macports.org" e-mail address.